azure-spring-boot-starter-parent/pom.xml

```diff
@@ -137,7 +137,7 @@
                 </dependencies>
                 <executions>
                     <execution>
-                        <phase>verify</phase>
+                        <phase>compile</phase>
```

Comment:
findbugs and cobertura-integration-test has conflict because of Cobertura 2.7 alters project.build.outputDirectory in order to produce report for integration tests (new kind of report introduced in 2.7). In the next step Findbugs checks violations not on just compiled classes but on instrumented. https://stackoverflow.com/questions/29594445/cobertura-maven-plugin-conflicts-with-findbugs
this change is to run findbug at compile phase instead of verify to workaround the issue.